RevenueCat/Scripts/Purchases.cs

```diff
@@ -171,113 +173,104 @@ public bool IsAnonymous()
 
         public void CheckTrialOrIntroductoryPriceEligibility(string[] productIdentifiers)
         {
-        
-        }
-    }
 
-    /*
-     * PurchaserInfo encapsulate the current status of subscriber. 
-     * Use it to determine which entitlements to unlock, typically by checking 
-     * ActiveSubscriptions or via LatestExpirationDate. 
-     * 
-     * Note: All DateTimes are in UTC, be sure to compare them with 
-     * DateTime.UtcNow
-     */
-    [SuppressMessage("ReSharper", "UnusedMember.Global")]
-    public class PurchaserInfo
-    {
-        private readonly PurchaserInfoResponse _response;
-
-        public PurchaserInfo(PurchaserInfoResponse response)
-        {
-            _response = response;
         }
 
-        public EntitlementInfos Entitlements
+        public void InvalidatePurchaserInfoCache()
```

Comment:
does the noop wrapper _have_ to live in this file? the others live in separate files, and moving it would reduce noise on this file. 
Since there's already a defined interface, the noop wrapper doesn't add much when reading

Reply:
No. It's part of the class extraction plan. I didn't want to do it in this PR because it would be hard to see the changes